src/ui/public/styles/variables/depth.less

```diff
@@ -0,0 +1,5 @@
+/**
+ * 1. The local nav contains tooltips which should pop over the filter bar.
+ */
+@globalDepthFilterBar: 2; /* 1 */
+@globalDepthLocalNav: 3; /* 1 */
```

Comment:
> Move the filter_bar directory into the kbn_top_nav directory, because the two are tightly coupled for most of our apps.

I don't think this is true, at least from a JS perspective. The filter_bar is a directive that can be used anywhere, not just in the kbn_top_nav. We definitely don't want to couple it to the `kbn_top_nav`, because each app needs the ability to conditionally render that directive depending on which query language is selected once [this PR](https://github.com/elastic/kibana/pull/11915) is merged. The query bar code itself doesn't even live in the kbn_top_nav, it's just in the transclude slot.